src/Billing/Services/Implementations/PaymentSucceededHandler.cs

```diff
@@ -141,6 +149,22 @@ await _referenceEventService.RaiseEventAsync(new ReferenceEvent
             await _organizationService.EnableAsync(organizationId.Value, subscription.CurrentPeriodEnd);
             var organization = await _organizationRepository.GetByIdAsync(organizationId.Value);
 
+            var users = await _organizationUserRepository.GetManyByOrganizationAsync(organizationId.Value,
+                OrganizationUserType.Owner);
+            foreach (var user in users)
+            {
+                var payload = new OrganizationStatusPushNotification
+                {
+                    UserId = user.UserId.Value,
+                    OrganizationId = organizationId.Value,
+                    Enabled = true
+                };
+
+                await _pushNotificationService.SendPayloadToUserAsync(payload.UserId.ToString(),
```

Comment:
I don't know why exactly we expose those methods, but non-selfhosted SignalR does not implement it https://github.com/bitwarden/server/blob/main/src/Core/Services/Implementations/AzureQueuePushNotificationService.cs#L211
In most cases we implement the method for specific notification in IPushNotificationService and all it's implementations.